contracts/reputationMiningCycle/ReputationMiningCycleCommon.sol

```diff
@@ -31,6 +32,20 @@ contract ReputationMiningCycleCommon is ReputationMiningCycleStorage, PatriciaTr
   // in reputationMiningCycleRespond. If you change one, you should change the other.
   uint256 constant MINING_WINDOW_SIZE = 60 * 60 * 1; // 1 hour
 
+  function getMinerAddress() internal view returns (address) {
```

Comment:
I would prefer this function return `msg.sender` in the event of no delegator (seems like the "right" behavior, since this isn't the place to assert whether the miner is staked), but I don't feel strongly enough to hold up the review.

Reply:
The implication here is that we'd have a `checkStaked` function, or similar, which would then be called after this function, and does exactly the same thing (including multiple calls to other contracts), but returns a boolean instead?

Would a name change placate you?

Reply:
Sure, `getMinerAddressIfStaked` would be better.